src/main/java/services/FruitServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package services;
+
+import dao.ProductDao;
+import java.util.List;
+import services.transaction.ProductTransactionMapper;
+import services.transaction.TransactionService;
+import services.transaction.TransactionServiceImpl;
+import services.transaction.model.ProductTransaction;
+
+public class FruitServiceImpl implements FruitService {
+    private final FileReaderService reader;
+    private final WriterService writerService;
+    private final ProductTransactionMapper unboxer;
+    private final OperationStrategy strategy;
+    private final ProductDao productDao;
+    private List<String> allTransactionsStr;
+    private List<ProductTransaction> productTransactions;
```

Comment:
Nikita meant only the transaction service, these variables you have to create in the method